### PR TITLE
Buepf video

### PR DESCRIPTION
**Pull request by gnurbs**
grosser abschnitt auf der homepage fürs video + stand, text von sabrina, während das video dort ist kein ccc-ch logo (quasi-stattdessen das video)

### CODE REVIEW

**Comment by dbrgn** [comment]
Bin dran.

File: content/media/buepf/test_en.srt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿1
+00:00:00,120 --> 00:00:01,620
+This is a test video,
```
Comment (dbrgn) [inline content/media/buepf/test_en.srt]: Ich nehme an das Testvideo kann ich wieder entfernen.

**Comment by gnurbs** [comment]
the subtitles atm do nothing (at least in firefox) and need to be converted to vtt - I would not delay deployment because of this

**Comment by dbrgn** [comment]
> the subtitles atm do nothing (at least in firefox) and need to be converted to vtt - I would not delay deployment because of this

I think that's because the development server is single threaded and cannot serve video and subtitle files at the same time.

**Comment by dbrgn** [comment]
Ari hat's abgesegnet, daher - merge & deploy! :)

**Pull request merged**